.github/workflows/build.yaml

```
@@ -0,0 +1,33 @@
+name: build
+
+on:
+  push:
+    branches:
+      - "main"
+    tags:
+      - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  conda-build:
+    needs: check-style
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Python build
+        run: "ci/build_python.sh true"
```

For consistency with the other command, use `./` here too (or delete below).
```suggestion
        run: "./ci/build_python.sh true"
```